### DISCUSSION TITLE
使用 Linear 来管理你的项目 - forecho's Blog

### DISCUSSION THREAD

**Discussion by giscus** [forecho/hugo-blog#41] [Announcements]
# 使用 Linear 来管理你的项目 - forecho's Blog

引言
互联网时代，大家做软件开发通常都是使用敏捷开发的模式，那么告别小作坊式的软件开发应该从使用项目管理工具开始。
从事 WEB 开发接近 9 年了，这些年来从刚开始的公司没有项目管理工具，到后来用过 Trello、TAPD 、Teambition 、Github Projects 、Jira 等不计其数的产品。

https://blog.forecho.com/using-linear.html

**Comment by woodwm** [comment]
我尝试用了一下，有点问题，Linear 的 issue 和 Github 的 issue 是差不多的，但Linear 没有提供同步的工具，这个意思是需要用 Linear 的 issue 取代 github 的 issue 吗？有一个同步的工具，但总感觉怪怪的，有些东西还是没法同步。

**Reply by forecho** [reply]
我不用 Github 的 Issue
